integration/common_integration_test.go

```diff
@@ -176,7 +176,7 @@ func (s *testState) newPostRequest(url string, body io.Reader) *http.Request {
 	return req
 }
 
-func (s *testState) newRequest(url string) *http.Request {
+func (s *testState) newGetRequest(url string) *http.Request {
```

Comment:
Can you rebase your changes? This refactor isn't compatible with the latest commits.